docs/src/bot.md

```diff
@@ -0,0 +1,148 @@
+# SnoopCompile Bot
+
+You can use SnoopCompile bot to automatically and continuously create precompile files.
+
+One should add 3 things to a package to make the bot work:
+
+----------------------------------
+
+
+- workflow file:
+
+create a workflow file with this path in your repository `.github/workflows/SnoopCompile.yml` and use the following content:
+
+```yaml
+name: SnoopCompile
+
+on:
+  - push
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['nightly']
+        julia-arch: [x64]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+      - name: Install dependencies
+        run: julia --project=@. -e 'using Pkg; Pkg.instantiate();'
```

Comment:
I think this can just be 

```suggestion
        run: julia --project -e 'using Pkg; Pkg.instantiate();'
```

Reply:
are these the same? 🤔

Reply:
No argument to `--project?  should use the project file in the current (or parent of current) working directory.